persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/TestEntity.scala

```diff
@@ -43,9 +43,9 @@ object TestEntity {
     override def aggregateTag: AggregateEventShards[Evt] = Evt.aggregateEventShards
   }
 
-  final case class Appended(element: String) extends Evt
+  final case class Appended(entityId: String, element: String) extends Evt
```

Comment:
the `entityId` is not exactly the same as the `persistenceId`, we prefix the `persistenceId` with the entity type name (see `PersistentEntityActor`) so we would have to define a separator and do string splitting. Not so nice, but I think the feature is important.

This probably means that it can't be introduced in the javadsl, because we can't change the format of the persistenceId.